apps/api/src/common/env/env.schema.ts

```diff
@@ -47,11 +53,23 @@ const devSchema = z.object({
   GITLAB_CLIENT_SECRET: z.string().optional(),
   GITLAB_CALLBACK_URL: z.string().optional(),
 
-  SENTRY_DSN: z.string().optional(),
+  SENTRY_DSN: z
+    .string()
+    .url({ message: 'SENTRY_DSN must be a valid URL' })
+    .optional(),
   SENTRY_ORG: z.string().optional(),
   SENTRY_PROJECT: z.string().optional(),
-  SENTRY_TRACES_SAMPLE_RATE: z.string().optional(),
-  SENTRY_PROFILES_SAMPLE_RATE: z.string().optional(),
+  SENTRY_TRACES_SAMPLE_RATE: z.string().optional().refine(validateSampleRate, {
+    message:
+      'SENTRY_TRACES_SAMPLE_RATE must be a string representing a number between 0 and 1'
+  }),
+  SENTRY_PROFILES_SAMPLE_RATE: z
+    .string()
+    .optional()
+    .refine(validateSampleRate, {
```

Comment:
You can try transforming the value into float. Look up the `transform` function in zod

Reply:
done